unity-renderer/Assets/DCLPlugins/ECS7/TestsUtils/ECS7TestEntity.cs

```diff
@@ -11,6 +11,7 @@
  */
 public class ECS7TestEntity : IDCLEntity
 {
+    public IParcelScene scene { get; set; }
```

Comment:
why do we need this reference here?

Reply:
Why wouldn't we? isn't the TestEntity simulating the real `DecentralandEntity`? `DecentralandEntity` has that, so if we want to be able to run test stuff with test entities instead of real entities, better to have that.

Reply:
so we can keep track of sdk7 dependency on the entity class, since we can currently refactor to move away from that entity class we at least have the tests failing to avoid using stuff that won't be valid for sdk7.
for example, that `scene` property (and like 70% of that class) is not needed in any part of the sdk7 code

Reply:
cool, done